src/publish.proj

```diff
@@ -65,4 +66,23 @@
                     ManifestBranch="$(ManifestBranch)"
                     ManifestCommit="$(ManifestCommit)" />
   </Target>
-</Project>
\ No newline at end of file
+
+
+  <Target Name="PublishAllSymbols" 
+          DependsOnTargets="SetupPublishSymbols;PublishSymbols">
+  </Target>
+
+  <Target Name="SetupPublishSymbols">
+    <PropertyGroup>
+      <ConvertPortablePdbsToWindowsPdbs>true</ConvertPortablePdbsToWindowsPdbs>
```

Comment:
Probably worth making this overridable if it is set already don't reset it.

Reply:
Done.